eagle-xl/scripts/gas_smoothing_lengths.py

```diff
@@ -0,0 +1,125 @@
+"""
+Makes a histogram of the gas smoothing lengths. Uses the swiftsimio library.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from swiftsimio import load
+from unyt import unyt_quantity, dimensionless, Mpc
+
+h_bounds = [1e-2, 5e2]  # comoving kpc
```

Comment:
@MatthieuSchaller are we sure we want the same bounds for XL?

Reply:
We use 500kpc as our max h so technically no need to go bigger. I have nevertheless increased the range a bit so the the max is seen.